Expensify/App
# Fix: [Android] Opening camera/gallery does not work on Android 11

### Details
Update the image picker and document picker libraries to their latest versions
Migrate usages to match the newer version specs

### Fixed Issues

Fixes #2531

### Tests
Try for both Android 10, 11 and iOS, desktop/web implementation haven't been changed but you can test that as well

1. Open a chat
2. Press the (+) button and select "Add Attachment" 
3. Try all the 3 options - Take Photo | Gallery | Documents
  - selecting photos documents should work
  - cancelling from take photo or the other options should not produce errors
  - changing permissions to "deny" using the camera should handle the error gracefully - see the videos bellow

### QA Steps
Same as above

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/12156624/121575360-f5244280-ca2f-11eb-8454-8391e18e0e26.mp4

#### Mobile Web


https://user-images.githubusercontent.com/12156624/121575767-69f77c80-ca30-11eb-82e6-875f21b5651b.mov

#### Desktop


https://user-images.githubusercontent.com/12156624/121575084-a1195e00-ca2f-11eb-9552-7274026bd4f3.mov

#### iOS


###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121574555-0e78bf00-ca2f-11eb-9808-8d05f5f1f814.mp4

###### Selecting a file from gallery or documents
Uploading Screen Recording 2021-06-10 at 21.47.51.mov…

#### Android


##### Android 11

###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121572297-ade88280-ca2c-11eb-8ebb-f8275f806fb2.mp4

###### Selecting a file from gallery or documents
https://user-images.githubusercontent.com/12156624/121572413-d3758c00-ca2c-11eb-8a01-c35b56dd6593.mp4

##### Android 10

###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121571078-3d8d3180-ca2b-11eb-9953-6fbab428ef36.mp4

###### Selecting a file from gallery or documents
https://user-images.githubusercontent.com/12156624/121570901-061e8500-ca2b-11eb-871b-64be01ebf3fa.mp4
